### DISCUSSION TITLE
API to insert nested fields (many to many) [May 04]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2019] [Discord Threads]
> 🙋‍♂️ **Nils**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Not at the moment. It will have to be done individually. However, this is something we will introrduce going forward  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/968862847197138954)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)

**Comment by MGW-SumathyV** [comment]
Hi,

I have three tables. Table1 is parent table.Table2 is also a parent table. 
In Table 3 I have created two LTAR columns ( many to many relation) referring to Table1 column and Table2 column.
How to add record to Table3 ? I tired via rest API its inserting empty values in the LTAR column.

Thanks and Regards,
Sumathy Venugopal
